tests/k8s/tests/02-cnp-specs.sh

```diff
@@ -152,11 +152,11 @@ echo "SUCCESS!"
 
 set +e
 
-kubectl delete -f "${bookinfo_dir}/"
+k8s_apply_policy $NAMESPACE delete "${bookinfo_dir}/"
 
 echo "Policies not found error is expected"
 
-kubectl delete -f "${bookinfo_dir}/policies"
+k8s_apply_policy $NAMESPACE delete "${bookinfo_dir}/policies"
```

Comment:
I think we should add a test that checks to make sure that no calls to `kubectl create` or `kubectl delete` are made outside of `tests/helpers.bash` - this will ensure that people use our helper functions that wait for desired state. This doesn't have to be done as part of this ticket. What do you think about this? I can create a separate issue for this.

Reply:
I think that's a good idea. Can you file an issue to track this for 0.12?

Reply:
Done: https://github.com/cilium/cilium/issues/1383